src/ralph/parsers.py

```diff
@@ -36,45 +37,31 @@ class GELFParser(BaseParser):
     documentation: https://docs.graylog.org/en/latest/pages/gelf.html
     """
 
-    def parse(self, input_file, filters=None, chunksize=50000):
+    name = "gelf"
+
+    def parse(self, input_file, chunksize=50000):
```

Comment:
According to the docstring the recommend benchmarked chunksize is <10k. Maybe this default should be adjusted?

Reply:
ditto